omero/sysadmins/cli/index.txt

```diff
@@ -13,6 +13,7 @@ need.
     config
     admin
     usergroup
+    data-ownership
```

Comment:
FWIW I was just following the pattern in `developers/Server/` so there may be a more general issue here.